test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs

```diff
@@ -2575,6 +2575,37 @@ public void ValidateJWE(JwtTheoryData theoryData)
             TestUtilities.AssertFailIfErrors(context);
         }
 
+        [Fact]
+        public async Task ValidateJsonWebTokenClaimMapping()
+        {
+            const string token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwidW5pcXVlX25hbWUiOiJNeVRlc3RVc2VyIiwiaWF0IjoxNTE2MjM5MDIyfQ.RdBPDWUy92hIn9MjxdzZjkn5yeFObc9-cq5xMaDFuag";
+
+            string symmetricKeyEncoded_256 = "Vbxq2mlbGJw8XH+ZoYBnUHmHga8/o/IduvU/Tht70iE=";
```

Comment:
Instead of creating a key, use KeyingMaterial.
There are a number of keys available for example: [DefaultSymmetricSecurityKey_256 ](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/2b1b78bf38a5739719e95b7045d81902a2cf09d5/test/Microsoft.IdentityModel.TestUtils/KeyingMaterial.cs#L232)